### ISSUE TITLE
H.265 HEVC Version ✔

### ISSUE THREAD

**Issue by masterofobzene** [Tichau/FileConverter#279]
Hi, I just wanted to share a version that I edited to support H.265 codecs.
https://github.com/masterofobzene/FileConverter-H265/releases/tag/1.2.4

There are 2 versions: CPU and GPU.
If you have a Nvidia card that supports hardware encoding, you may want to use the GPU version for a real speed boost.

Take care.

**Comment by tyeeman** [comment]
Just tried your h265 gpu version and I set crf to 18 and ffmpeg command line is using crf 32! Please check at your end!

**Comment by tyeeman** [comment]
Yes, I was looking at the "Log" tab when the encode finishes. The full output of ffmpeg is there with the command line used. Here is a sample I just did- Unfortunately, that window can't be copied so here are 2 screenshots showing the command line. 


<img width="600" alt="2022-10-22_085729" src="https://user-images.githubusercontent.com/4469306/197349139-34e580eb-ba0d-474c-a5d3-2f29ba955fa3.png">
<img width="600" alt="2022-10-22_085742" src="https://user-images.githubusercontent.com/4469306/197349147-1f9db29d-ef2e-4e42-9d35-bb06321f4435.png">

There are lots of articles about which commands to use for the gpu. It would be nice that I could just paste my personal choices into a new "User Commands" field.

**Comment by Toei79** [comment]
question its any way to add support for .ts file for now i need do change manually .ts to .mkv to make a conversion thanks!

**Comment by Toei79** [comment]
> > question its any way to add support for .ts file for now i need do change manually .ts to .mkv to make a conversion thanks!
> 
> Not sure what you mean, the release on my fork has .ts support.

You looks like what I mean thanks , i try yours

On this one you don't have .ts option

**Comment by Toei79** [comment]
Uh

**Comment by Toei79** [comment]
> Mmm maybe you are a little confused... I only forked this app and modified it to support .ts and h265 codecs. This repo "Tichau/FileConverter" is the original project. Mine is "masterofobzene/FileConverter-H265" , a "patched" version. Sorry I didn't add it in the OP (was a later modification)

i know you forked app , i know what yo do there,  i know original project its tichau, file convert, etc etc.  
i already know all that.

**Comment by ifelixculpa** [comment]
> Mmm maybe you are a little confused... I only forked this app and modified it to support .ts and h265 codecs. This repo "Tichau/FileConverter" is the original project. Mine is "masterofobzene/FileConverter-H265" , a "patched" version. Sorry I didn't add it in the OP (was a later modification)

i am using ur work mate and it works flawlessly...could you please add it to chocolatey packages

**Comment by inuonly** [comment]
> GPU

how to access settings , in the app list it only shows fie converter but no file converter settings.

**Comment by Tichau** [comment]
Hey,
I finally had some time to prepare and finish the release of a version of File Converter.
Thanks for creating this fork and giving alternative to people that needed these features <3

I'll be happy to merge your changes if you send me a pull request.
